src/en/guide/conf/config/logging/externalLoggingConfiguration.adoc

```diff
@@ -4,18 +4,18 @@ If you set the configuration property `logging.config`, you can instruct `Logbac
 .grails-app/conf/application.yml
 ----
 logging:
-    config: /Users/me/config/logback.groovy
+    config: /Users/me/config/logback.xml
```

Comment:
Should these 3 be logback-spring.xml?

Reply:
I guess we can use any filename as an external file since we are explitily pointing to it, but I agree we can use `logback-spring.xml`